projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -540,12 +542,16 @@ class="launch-bar-global-styles-upgrade"
 					class="launch-bar-global-styles-reset"
 					href="https://wordpress.com/support/using-styles/#reset-all-styles"
 					target="_blank"
+					data-blog-id="<?php echo esc_attr( (string) $blog_id ); ?>"
+					data-global-styles-id="<?php echo esc_attr( (string) $global_styles_id ); ?>"
```

Comment:
This is a first pass, but thinking of alternatives to exposing blog and global styles id to frontend.

Via a query e.g., `?reset-global-styles=true`? Then the PHP could do all the work, and also double check that the user is logged in, has privileges etc.

If it fails for whatever reason, then we could render the link to the help docs.